src/main/java/lesson02/tasksresolve/SumApp.java

```diff
@@ -0,0 +1,33 @@
+package lesson02.tasksresolve;
+/*
+Task 7 (SumApp)
+Посчитать сумму целых чисел, которые пользователь вводит с консоли. Считывать числа, пока не будет введено 0.
+Считать, что пользователь вводит только корректные числа типа int.
+Ограничения: нет.
+int clientElemsSum()
+Входные аргументы: Чтение с консоли
+Тип результата int
+ */
+
+import lesson02.utility.ConsoleInput;
+
+public class SumApp {
+    private static final String MESSAGE_FOR_INPUT = "Please input integer number: ";
+
+    public static void main (String[] args) {
+        clientElemsSum();
+    }
+
+    public static int clientElemsSum (){
+        int sumOfInputNumbers = 0;
+        for (int i = 1; i > 0; i++) {
```

Comment:
"For" - is usually used when you need to iterate over a fixed amount of items.
While - is more sufficient in this case.

Reply:
I used "For" loop because there is a requirement that we need to break the loop once the input number is equal to 0. Since I'm calling number input from the console within the loop I'm not able to use (inputNumber!=0) condition in "while" statement.